database/src/androidTest/java/com/firebase/ui/database/FirebaseArrayOfObjectsTest.java

```diff
@@ -35,54 +33,46 @@
 import static com.firebase.ui.database.TestUtils.runAndWaitUntil;
 
 @RunWith(AndroidJUnit4.class)
-@SmallTest
-public class FirebaseArrayOfObjectsTest extends InstrumentationTestCase {
+public class FirebaseArrayOfObjectsTest {
+    private static final int INITIAL_SIZE = 3;
+
     private DatabaseReference mRef;
     private FirebaseArray mArray;
 
     @Before
     public void setUp() throws Exception {
-        FirebaseApp app = getAppInstance(getInstrumentation().getContext());
-        mRef = FirebaseDatabase.getInstance(app).getReference()
-                .child("firebasearray").child("objects");
+        FirebaseApp app = getAppInstance(InstrumentationRegistry.getContext());
+        mRef = FirebaseDatabase.getInstance(app)
+                .getReference()
+                .child("firebasearray")
+                .child("objects");
         mArray = new FirebaseArray(mRef);
         mRef.removeValue();
         runAndWaitUntil(mArray, new Runnable() {
-                            @Override
-                            public void run() {
-                                for (int i = 1; i <= 3; i++) {
-                                    mRef.push().setValue(new Bean(i, "Text " + i, i % 2 == 0), i);
-                                }
-                            }
-                        }, new Callable<Boolean>() {
-                            @Override
-                            public Boolean call() throws Exception {
-                                return mArray.getCount() == 3;
-                            }
-                        }
-        );
+            @Override
+            public void run() {
+                for (int i = 1; i <= INITIAL_SIZE; i++) {
+                    mRef.push().setValue(new Bean(i, "Text " + i, i % 2 == 0), i);
+                }
+            }
+        }, new Callable<Boolean>() {
+            @Override
+            public Boolean call() throws Exception {
+                return mArray.getCount() == INITIAL_SIZE;
+            }
+        });
     }
 
     @After
     public void tearDown() throws Exception {
-        if (mRef != null) {
-            mRef.getRoot().removeValue();
-        }
-
-        if (mArray != null) {
-            mArray.cleanup();
-        }
-    }
-
-    @Test
-    public void testSize() throws Exception {
-        assertEquals(3, mArray.getCount());
+        mArray.cleanup();
+        mRef.getRoot().removeValue();
     }
 
     @Test
     public void testPushIncreasesSize() throws Exception {
-        assertEquals(3, mArray.getCount());
```

Comment:
@CodingDoug removed it in his PR. If you'd like I'll add it back (I'm not sure why he removed it).